task/conftest-clair/0.1/conftest-clair.yaml

```diff
@@ -36,5 +36,32 @@ spec:
                    failures: (.failures // [])|map(.metadata.details.name) | unique
                  }' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json || true)
         echo "${HACBS_TEST_OUTPUT:-${HACBS_ERROR_OUTPUT}}" | tee $(results.HACBS_TEST_OUTPUT.path)
-  workspaces:
-    - name: conftest-ws
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
```

Comment:
it should be merged on step level, there should not be multiple tasks in one task dir.